frontend/libs/studio-components/src/components/StudioTextfield/StudioTextfield.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { forwardRef } from 'react';
+import type { Ref, ReactElement, HTMLAttributes } from 'react';
+import classes from './StudioTextfield.module.css';
+import { Textfield } from '@digdir/designsystemet-react';
+import type { TextfieldProps } from '@digdir/designsystemet-react';
+import { StudioTag } from '../StudioTag';
+
+export type StudioTextfieldProps = TextfieldProps & {
+  requiredText?: string;
+};
+
+function StudioTextfield(
+  { children, required, requiredText, label, ...rest }: StudioTextfieldProps,
+  ref: Ref<HTMLInputElement>,
+): ReactElement {
+  // Designsystemet has conditional types, so if we extract label from props, we must
+  // check if the usage has aria-labelledby or aria-label and if true not use the label.
+  if (hasAriaLabelledBy(rest) || hasAriaLabel(rest)) {
+    return <Textfield ref={ref} {...rest} />;
+  }
+
+  const labelComponent = (
+    <RequiredWrapper required={required} requiredText={requiredText}>
+      {label}
+    </RequiredWrapper>
+  );
+
+  return <Textfield ref={ref} {...rest} label={labelComponent} />;
```

Comment:
Should we use the JSX component directly instead of creating a ReactNode explicitly? It’s cleaner and more idiomatic in React. 😊 
```suggestion
  const labelComponent = (
    <RequiredWrapper required={required} requiredText={requiredText}>
      {label}
    </RequiredWrapper>
  );

  return <Textfield ref={ref} {...rest} label={<LabelComponent required={required} requiredText={requiredText}>{label}</LabelComponent >} />;
```

Reply:
Agree 😄 
Fixed now!